Azure/azure-windows/_meta/fields.yml

```diff
@@ -0,0 +1,81 @@
+azure_windows.event_data:
+  name: azure_windows.event_data
+  description: "The event-specific data"
+  type: object
+
+azure_windows.opcode:
+  name: azure_windows.opcode
+  description: "The opcode defined in the event. Task and opcode are typically used to identify the location in the application from where the event was logged."
+  type: keyword
+
+azure_windows.provider_guid:
+  name: azure_windows.provider_guid
+  description: "A globally unique identifier that identifies the provider that logged the event"
+  type: keyword
+
+azure_windows.provider_name:
+  name: azure_windows.provider_name
+  description: "The source of the event log record (the application or service that logged the record)."
+  type: keyword
+
+azure_windows.task:
+  name: azure_windows.task
+  description: "The task defined in the event. Task and opcode are typically used to identify the location in the application from where the event was logged"
+  type: keyword
+
+azure_windows.user.domain.name:
+  name: azure_windows.user.domain.name
+  description: "The domain that the account associated with this event is a member of"
+  type: keyword
+
+azure_windows.user.identifier:
+  name: azure_windows.user.identifier
+  description: "The Windows security identifier (SID) of the account associated with this event"
+  type: keyword
+
+azure_windows.user.name:
+  name: azure_windows.user.name
+  description: "Name of the user associated with this event"
+  type: keyword
+
+azure_windows.user.type:
+  name: azure_windows.user.type
+  description: "The type of account associated with this event"
+  type: keyword
+
+
+###############################################################
+#                                                             #
+# next fields are declared for backward compatibility purpose #
+#                                                             #
+###############################################################
+
+action.properties:
+  name: action.properties
+  description: "A list of objects"
+  type: text
+
+action.target:
+  name: action.target
+  description: "The target of the action"
+  type: keyword
+
+process.ppid:
+  name: process.ppid
+  description: "The identifier of the parent of the process"
+  type: number
+
```

Comment:
Already exists in ECS https://www.elastic.co/guide/en/ecs/1.7/ecs-process.html#field-process-ppid
```suggestion
```

Reply:
In the last versions of the specification, `process.ppid` was replaced by `process.parent.pid`.
I kept this field in the taxonomy for backward compatibility. However, after analyzing the detection rules, it seems that field is not used (so I removed it from the format).